README.md

```diff
@@ -13,60 +15,39 @@ Android Image Cropper
 
 [See GitHub Wiki for more info.](https://github.com/CanHub/Android-Image-Cropper/wiki)
 
+#### Step 1. Add the JitPack repository to your root build.gradle
 
-**The library is release with github Packages, if you already have github packages libraris can jump to second step**
-### 1. Generate a Personal Access Token for GitHub
-Reference: [Github Personal Access Token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token)
```

Comment:
Should we keep both? with the GitPackages ? 

If not, how this could affect who is already using gitPackages?

Reply:
If we decide to not keep, please delete `publish.gradle` file

Reply:
I thought better, let's remove the Github Package way if we don't spot any issue for who is using it

Reply:
I believe it's fine for anyone who currently was using gitPackages. They would just need to add the jitpack repo to their project gradle and update the dependency and all should be fine. I will delete the `publish.gradle` next commit 👍🏻